src/pages/VenuePage/VenuePage.tsx

```diff
@@ -132,10 +135,22 @@ export const VenuePage: React.FC = () => {
 
   // @debt refactor how user location updates works here to encapsulate in a hook or similar?
   useEffect(() => {
-    if (!userId || !venueName) return;
+    if (!userId || !venueName || !venueId) return;
+
+    const getWholePath = async () => {
+      const { sovereignVenue, checkedVenueIds } = await fetchSovereignVenue(
+        venueId
+      );
+
+      const allVenues = [...checkedVenueIds, sovereignVenue.id].reverse();
+
+      const locationPath = wrapIntoSlashes(allVenues.join("/"));
+
+      setLocationData({ userId, locationPath });
```

Comment:
This  is the only place that uses `setLocationData`, which in turn just calls `updateLocationData`.
https://github.com/sparkletown/sparkle/blob/7f8e625a2b39dd97ac7eb2d09a8b1e188da3101a/src/utils/userLocation.ts#L34-L42
Can we just call `updateLocationData` and remove `setLocationData` from the codebase? It's really confusing.

Similarly, we can remove `updateCurrentLocationData`

Reply:
Done in https://github.com/sparkletown/sparkle/pull/2202/commits/dd77d4b9afdba40f55a14bd7da7e623874e9e92c